src/gui/menu_manager.cpp

```diff
@@ -343,7 +375,7 @@ MenuManager::previous_menu() const
 }
 
 void
-MenuManager::transition(Menu* from, Menu* to)
+MenuManager::transition(Menu* from, Menu* to, bool call_this)
```

Comment:
Is this parameter needed?

Reply:
WIthout this parameter, calls to the `transition()` function that may include `nullptr`, appear to be ambiguous.

Reply:
Alright, also yeah, adding comment describing that is a good idea.